.travis.yml

```diff
@@ -14,33 +13,24 @@ branches:
     - master
 
 env:
-  matrix:
-   - ''  # main tests
-   - SAMPLE_APP=1  # sample app tests
+  - DJANGO="django>=2.2,<3.0" # main tests
+  - DJANGO="django>=3.0,<3.1"
+  - DJANGO="django>=2.2,<3.0" SAMPLE_APP=1 # sample app tests
+  - DJANGO="django>=3.0,<3.1" SAMPLE_APP=1
 
 before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - npm install -g jslint
   - jslint ./django_ipam/static/django-ipam/js/*.js
 
 install:
-  - pip install "django>=2.0,<2.3"
+  - pip install $DJANGO
   - python setup.py -q develop
 
 script:
   - >
     openwisp-utils-qa-checks --migration-path "./django_ipam/migrations/ ./tests/sample_ipam/migrations/"
     --skip-checkmakemigrations
```

Comment:
you either should remove `--skip-checkmakemigrations` or you should not remove the check below (just remove the if, not the check).

Reply:
Okay, I'll correct that, but could you tell me why was that check executed previously only for one build, that with python 3.5? I don't get it.

Reply:
Because 2.7 on travis had some issue if I recall correctly! :smile: